README.md

```diff
@@ -357,6 +358,27 @@ To substitute Address data types with a fixed json fragement
 }
 ```
 
+To perturb age fields of Condition resource by sampling noise from a fixed span
+```json
+{
+  "path": "Condition.onset | Condition.abatement as Age",
+  "method": "perturb",
+  "span": 3,
+  "rangeType": "fixed",
+  "roundTo": 0
+}
+```
+To perturb age fields of Condition resource by sampling noise from a proportional span

```

Comment:
Here too, provide numerical details for clarity.

Reply:
Thanks, updated